docs/gradle-plugin/index.md

```diff
@@ -0,0 +1,304 @@
+## Table of contents
+
+- [Features](#features)
+- [Quickstart](#quickstart)
+- [Using Kover tasks](#using-kover-tasks)
+  - [Kotlin JVM project](#kotlin-jvm-project)
+  - [Kotlin Android project](#kotlin-android-project)
+  - [Kotlin multiplatform project](#kotlin-multiplatform-project)
+- [Multiproject build](#multiproject-build)
+  - [Single report over several projects](#single-report-over-several-projects)
+- [Filtering reports](#filtering-reports)
+- [Kover configuration](configuring)
+- [Kover Tasks](#kover-tasks)
+  - [Kover default tasks](#kover-default-tasks)
+  - [Kover Android tasks](#kover-android-tasks)
+- [Instrumentation](#instrumentation)
+- [Using JaCoCo](#using-jacoco)
+- [Implicit plugin dependencies](#implicit-plugin-dependencies)
+- [Gradle Plugin DSL docs](dokka)
+
+## Features
+
+* Collection of code coverage through `JVM` test tasks.
+* `HTML` and `XML` reports.
+* Support for `Kotlin/JVM`, `Kotlin Multiplatform` and mixed `Kotlin-Java` sources with zero additional configuration.
+* Support for `Kotlin Android` without the need to divide it into build types and flavours.
+* Verification rules with bounds to keep track of coverage.
+* Customizable filters for instrumented classes.
```

Comment:
Should this feature list be identical with one in the README?

Reply:
OK